client/js/app.js

```diff
@@ -561,6 +554,11 @@
         }
     }
 
+    function contain(min, max, value) {
+        return Math.min(max, Math.max(min, value));
```

Comment:
@bigfoot90  this basically "contains" the value inside the min-max range. If it's smaller than min, it returns min. If it's bigger than max, it returns max. This is to ensure "value" doesn't go out of the bounds specified.